synphot/tests/test_observation.py

```diff
@@ -22,6 +23,8 @@
 from ..models import (BlackBodyNorm1D, Box1D, ConstFlux1D, Empirical1D,
                       GaussianFlux1D)
 from ..observation import Observation
```

Comment:
You can import them all in one line. (Eh, not sure why GitHub diff is showing the removal of a leading whitespace; there should be none.)
```suggestion
from ..observation import Observation, howell_snr, exptime_from_howell_snr
```